src/test/java/com/imsweb/geocoder/GeocodingInputTest.java

```diff
@@ -35,7 +36,7 @@ public void testBasicInput() {
         assertEquals("false", queryParams.get("notStore"));
         assertEquals("true", queryParams.get("census"));
         assertEquals("allAvailable", queryParams.get("censusYear"));
-
+        assertEquals("true", queryParams.get("ShouldDoExhaustiveSearch"));
```

Comment:
Shouldn't all the query params use camelCase?  Others do, but there are a couple that do not.  I think we should be consistent.